ugvc/mrd/srsnv_inference_utils.py

```diff
@@ -126,7 +126,13 @@ def process_records(self, records: list[pysam.VariantRecord]) -> list[pysam.Vari
         """
         # Convert list of variant records to a pandas DataFrame
         columns = self.numerical_features + self.categorical_features
-        data = [{column: variant.info[column] for column in columns} for variant in records]
+        data = [
```

Comment:
I understand what you are trying to do here.
But I wonder if there is a simpler syntax.
Can we read the info columns like before, and then add ref and alt with simple apply/append?

Reply:
tried it, and seems this is the best way to create a dictionary from two dictionaries

Reply:
done